Assets/Scripts/GameHandler.cs

```diff
@@ -9,19 +9,21 @@ public class GameHandler : MonoBehaviour
     [SerializeField] public GameObject bulletPrefab;
     [SerializeField] private GameObject _playerPrefab;
     [SerializeField] private int _bulletDamage;
-    [SerializeField] private Color baseBulletColor;
+    [SerializeField] private Color _baseBulletColor;
 
-    public ISceneObject _player;
+    public ISceneObject player;
+
+    public Camera mainCam;
 
     private List<ISceneObject> _updateables = new List<ISceneObject>();
 
-    private Bullet _bullet;
-    
+    private Bullet _bullet;  
```

Comment:
Moet bullet in deze scope leven?
Volgens mij kan bullet beter worden verplaatst naar de CreateBullet() functie